sdk/core/Azure.Core.Experimental/src/DynamicData/DynamicDataOptions.cs

```diff
@@ -14,6 +14,11 @@ public class DynamicDataOptions
         };
         internal static DynamicDataOptions Default { get => _default; }
 
+        /// <summary>
+        /// Gets or sets an object that specifies how dynamic property names will be mapped to member names in the data buffer.
```

Comment:
nit: "in the data buffer" sounds too implementation detail-y to me. I wonder if we should change terminology slightly to something like "of the underlying data model".

Reply:
I feel like using the word model is tied to a subset of use cases as well, but I'm happy to rework this to remove the word buffer.